srcpkgs/composer8.0/template

```diff
@@ -13,6 +13,7 @@ distfiles="https://github.com/composer/composer/releases/download/${version}/com
  https://raw.githubusercontent.com/composer/composer/main/LICENSE"
 checksum="2e1061821951c6a5ece033a025d06296d4a1d056fee2f4bebd35815cf4b1b0f9
  7855ac293067aebe7e51afdd23b9dea54b8be24187dbecc9b9142581c37f596c"
+alternatives="composer:composer:/usr/bin/composer8.0"
```

Comment:
these packages need to be revbumped, otherwise people would still be getting the old version without alternatives until the next update

Reply:
Oops, must have accidentally dropped them when I was rebasing, fixed